### DISCUSSION TITLE
Expo SDK long term support

### DISCUSSION THREAD

**Discussion by jurica-penjgusic** [expo/expo#15870] [Archived: Questions] [answered]
Hi there!

We were wondering is Expo planning to support long term SDKs?
Keeping up with Expo team and upgrading project every 3 or 6 months can be challenging especially in enterprise world.

Tnx in advance!

**Comment by EvanBacon** [comment, marked as answer]
We're deprecating the expo-cli and moving the core commands into the expo package, that should fix most of the long term support issues when it's complete.

**Reply by jurica-penjgusic** [reply]
Sweet!
Keep with good work!

**Reply by raajnadar** [reply]
@EvanBacon can you link the PR?
